kubernetes-node-pool.tf

```diff
@@ -14,6 +14,7 @@ module "node_pool_blue" {
   project_id         = var.metal_create_project ? metal_project.new_project[0].id : var.project_id
   storage            = var.storage
 }
+ccm_enabled = var.ccm_enabled
```

Comment:
This line looks to be outside of any blocks.

`terraform validate` should show that this is not valid.

Reply:
Fixed